src/types/api.js

```diff
@@ -145,12 +145,13 @@
  * for the complete list and usage of each.
  *
  * @typedef SearchQuery
+ * @prop {number} [limit]
+ * @prop {number} [offset]
```

Comment:
Are there still cases in the client code that use `offset` instead of `search_after`?

Reply:
No. I'll remove this from the typedef.